unblob/processing.py

```diff
@@ -417,6 +418,12 @@ def _extract_chunk(self, file, chunk: ValidChunk):  # noqa: C901
             )
 
 
+def patch_chunks(chunks: Sequence[Chunk], file: File):
```

Comment:
`patch_chunks` is too generic as a function name, I would use something like `assign_file_to_chunks`

Reply:
done